examples/coap-server.c

```diff
@@ -282,6 +282,59 @@ hnd_get_time(coap_resource_t *resource,
   }
 }
 
+/* This is them as hnd_get_time(), but FETCH is used and the data is examined */
+static void
+hnd_fetch_time(coap_resource_t *resource,
```

Comment:
I suggest moving this enhancement to a separate commit within this PR. Also, I think the same handler function can be used for get and fetch where the location of the string "ticks" depends on the request method.

Reply:
Split out as suggested and code pushed.